README.md

```diff
@@ -91,9 +91,13 @@ For more details of the JavaScript model, please refer to [JavaScript module REA
 You can also format inputs on your terminal with `budoux` command.
 
 ```shellsession
-$ budoux 本日は晴天です。
+$ budoux 本日は晴天です。  # default: japanese
 本日は
 晴天です。
+
+$ budoux -l zh 今天天气晴朗。
```

Comment:
Actually, I want to use `zh-hans` as the language code because I would like to emphasize that this is for Simplified Chinese (hans), not for Traditional Chinese (hant) here.

Reply:
a52c8ab